kalitectl.py

```diff
@@ -480,12 +454,13 @@ def start(debug=False, daemonize=True, args=[], skip_job_scheduler=False, port=N
     if daemonize:
         
         from django.utils.daemonize import become_daemon
-        logfile = os.path.join(KALITE_HOME, "kalite.log") if os.environ.get("NAIVE_LOGGING", False) else None
-        if logfile:
-            print("With logging...")
-            become_daemon(out_log=logfile, err_log=logfile)
-        else:
-            become_daemon()
+        kwargs = {}
+        # Truncate the file
+        open(SERVER_LOG, "w").truncate()
+        print("Going to daemon mode, logging to {0}".format(SERVER_LOG))
+        kwargs['out_log'] = SERVER_LOG
+        kwargs['err_log'] = SERVER_LOG
```

Comment:
Seems to grow slow enough (about 50 bytes/second), but I'm seeing this error repeatedly in the logs:
```py
Traceback (most recent call last):
  File "C:\Python27\lib\logging\__init__.py", line 880, in emit
    stream.write(fs % msg)
ValueError: I/O operation on closed file
Logged from file _cplogging.py, line 280
```